packages/flutter_tools/test/integration/flutter_attach_test.dart

```diff
@@ -37,9 +37,8 @@ void main() {
     testUsingContext('can hot reload', () async {
       await _flutterRun.run(withDebugger: true);
       await _flutterAttach.attach(_flutterRun.vmServicePort);
-
       await _flutterAttach.hotReload();
     });
-    // Skip on Windows due to https://github.com/flutter/flutter/issues/17833
+    // SKIP(dantup): https://github.com/flutter/flutter/issues/17833.
```

Comment:
Not sure if we use this convention in the repo. I'd use `TODO(...)` unless you find another occurance of `SKIP`.

Reply:
Yeah, I made this up to try and make it easier for me to find them all when I periodically check whether we can unskip any. I've changed them to TODO though, they should still be easy to find, there aren't many!